src/sentry/models/organizationaccessrequest.py

```diff
@@ -66,6 +66,7 @@ def send_request_email(self):
 
         msg.send_async([m.user.email for m in member_list])
 
+    #
```

Comment:
what's this for?

Reply:
That shouldn't be there, let me clean up some weirdness in a new commit.